### DISCUSSION TITLE
How it work

### DISCUSSION THREAD

**Discussion by wadwin** [tmk/tmk_keyboard#679] [General]
Hello,
how does the project work ?
Once the arduino card has been programmed, is it possible to connect, for example, an android smartphone to the bluetooth chip to simulate a keyboard or a bluetooth mouse ?
Thx

**Comment by tmk** [comment]
This project doesn't support arduino and bluetooth that well. You can't compile this with arduino tool for example.  You may want to search other arduino projects.

There is no simple example project for buletooth in this repo, but these two projects.

This uses bluetooth module RN42-I/RM and I confirmed this can be used as bluetooth keyboard and mouse on android.
https://github.com/tmk/tmk_keyboard/tree/master/keyboard/hhkb

Another project supports bluetooth using adafruit Bluefruit EZ-Key.
https://github.com/tmk/tmk_keyboard/blob/master/orphan/terminal_bluefruit/Makefile
https://github.com/tmk/tmk_keyboard/tree/master/tmk_core/protocol/bluefruit

**Comment by wadwin** [comment]
Je recherche un projet pour simuler une souris Bluetooth par programmation open source and cheap,
is this the case for other projects that you send to me ?
